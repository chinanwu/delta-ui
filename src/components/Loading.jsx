import React from 'react';

import './Loading.less';

export const Loading = () => {
	return <div className="Loading">'ey I'm loadin' 'ere!</div>;
};

export default Loading;
