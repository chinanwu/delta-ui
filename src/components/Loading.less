.Loading {
	width: 100%;
	position: absolute;
	top: 0;
	text-align: center;
}
