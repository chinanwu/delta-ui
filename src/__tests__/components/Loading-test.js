import { shallow } from 'enzyme';
import React from 'react';

import { Loading } from '../../components/Loading.jsx';

jest.unmock('../../components/Loading.jsx');

describe('Loading component', () => {
	describe('rendering', () => {
		it('renders Loading', () => {
			const wrapper = shallow(<Loading />);
			expect(wrapper.getElement()).toMatchSnapshot();
		});
	});
});
