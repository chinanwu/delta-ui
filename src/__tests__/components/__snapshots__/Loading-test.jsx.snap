// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component rendering renders Loading 1`] = `
<div
  aria-busy="true"
  className="Loading"
  role="alert"
>
  <Loader
    color="#000000"
    css=""
    loading={true}
    size={35}
  />
</div>
`;

exports[`Loading component rendering renders NotFound 1`] = `
<div
  className="Loading"
>
  'ey I'm loadin' 'ere!
</div>
`;
